tasks/run_cd4pe_job.rb

```diff
@@ -270,7 +270,11 @@ def send_job_output_to_cd4pe(output)
     client = CD4PEClient.new(web_ui_endpoint: api_endpoint, job_token: @job_token, ca_cert_file: @ca_cert_file, logger: @logger)
 
     begin
-      client.make_request(:post, api_endpoint, payload.to_json)
+      response = client.make_request(:post, api_endpoint, payload.to_json)
+      if (response.code != 200)
+        @logger.log("Sending logs directly to CD4PE not supported in CD4PE version < 3.10.0. Printing logs to std out.")
+        puts output.to_json
+      end
     rescue => e
       @logger.log("Problem sending logs to CD4PE. Printing logs to std out. #{e.message}")
```

Comment:
While I understand that this hasn't changed, it's difficult to visually parse what the header vs the error message is.

Reply:
I'm not exactly sure what you mean by "header" vs error message. Are you saying the message `Sending logs directly to CD4PE not supported in CD4PE version < 3.10.0` is difficult to understand?

Reply:
No. I'm saying the old format error message that dumps `#{e.message}` with no delimiters to indicate its a discrete unit of info is hard to understand.

Reply:
So, this particular line of code prints a helpful message, then prints the entirety of the message contained inside the exception. What do you mean by "no delimiters to indicate it's a discrete unit of info?" What sort of delimeters? And what are you confused about being a discrete piece of info?

Reply:
There is no way to visually discern what is part of `#{e.message}` vs. what we are prefacing it with.

Reply:
Got it. Will add text to make it obvious that the second part is an error message